api/src/main/proto/stellarstation/api/v1/stellarstation.proto

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright 2018 Infostellar, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+syntax = "proto3";
+
+import "google/protobuf/timestamp.proto";
+
+package stellarstation.api.v1;
+
+option java_multiple_files = true;
+option java_outer_classname = "StellarstationProto";
+option java_package = "com.stellarstation.api.v1";
+
+// The public API service of Stellarstation, a system supporting communication between satellites
+// and groundstations. This API is for use by an operator of a satellite - it is illegal to specify
```

Comment:
Remove the phrase `it is illegal...`? Maybe you use the term "illegal" not as violation of laws but as invalid parameters?

Reply:
Indeed I was thinking `IllegalArgumentException`, but since this gRPC it was better to think of `INVALID_ARGUMENT` :)